docs/reference/contrib/routablepage.rst

```diff
@@ -30,7 +30,7 @@ Add ``"wagtail.contrib.routable_page"`` to your INSTALLED_APPS:
 The basics
 ==========
 
-To use ``RoutablePageMixin``, you need to make your class inherit from both :class:`wagtail.contrib.routable_page.models.RoutablePageMixin` and :class:`wagtail.models.Page`, then define some view methods and decorate them with ``wagtail.contrib.routable_page.models.route``. These view methods behave like ordinary Django view functions, and must return an ``HttpResponse`` object; typically this is done through a call to ``django.shortcuts.render``.
+To use ``RoutablePageMixin``, you need to make your class inherit from both :class:`wagtail.contrib.routable_page.models.RoutablePageMixin` and :class:`wagtail.models.Page`, then define some view methods and decorate them with ``wagtail.contrib.routable_page.models.path`` or ``wagtail.contrib.routable_page.models.re_path``. These view methods behave like ordinary Django view functions, and must return an ``HttpResponse`` object; typically this is done through a call to ``django.shortcuts.render``.
```

Comment:
I think we should mention somewhere where you would use `path` and where `re_path` - just a simple comparison to Django's `path` and `re_path` to be a bit more explicit would do I think, or a link.

Reply:
I've mentioned it below this section.